### PR TITLE
Celebrity problem | Graphs

### LINKED ISSUE #3577: Celebrity Problem in C++/CPP | Graphs | GSSoC'21
## 💥 Proposal

I will implement the celebrity problem in CPP/C++ as part of gssoc'21, under Graphs section 

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

YES

### PR DESCRIPTION
**Pull request by Harikrishnan6336**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES

### Description
Added Celebrity Problem in C++

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Fixes #3577

### CODE REVIEW

**Comment by Harikrishnan6336** [comment]
@bhav09 @anushkrishnav @siddharth25pandey Kindly review my PR

File: C-Plus-Plus/graphs/celebrity_problem.cpp [hunk 1]
```diff
@@ -0,0 +1,67 @@
+// C++ program to solve the Celebrity Problem
+#include <bits/stdc++.h>
+using namespace std;
+
```
Comment (anushkrishnav) [inline C-Plus-Plus/graphs/celebrity_problem.cpp:4]: Add a comment explaining what work does the function do , input it takes etc

File: C-Plus-Plus/graphs/celebrity_problem.cpp [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        if (person[low][i] == 1 or person[i][low] == 0)
+            return -1;
+    }
+    return low;
+}
+
+int main()
+{
+    int n;
+    cout << "How many person are present in the party? ";
```
Comment (anushkrishnav) [inline C-Plus-Plus/graphs/celebrity_problem.cpp]: Fix Grammatical error

File: C-Plus-Plus/graphs/celebrity_problem.cpp (hunk 1 above)
Comment (anushkrishnav) [inline C-Plus-Plus/graphs/celebrity_problem.cpp:15]: Add a comment explaining what work does the function do, input it takes, etc

**Review by anushkrishnav — CHANGES_REQUESTED** [review]

**Comment by anushkrishnav** [comment]
Fix the issues and run the code through a linter

**Comment by Harikrishnan6336** [comment]
@anushkrishnav I have made the required changes. Kindly review the PR

**Comment by anushkrishnav** [comment]
I still find linting issues on your code kindly use [CPPLint](https://cpplint.appspot.com/CppLint) and fix those linting issues.

**Comment by Harikrishnan6336** [comment]
@anushkrishnav Sorry, I wasn't aware of a linter.

I have solved every issue in the linter except two:
They are 
- No copyright message found. You should have a line: "Copyright [year] <Copyright Owner>"
-  Do not use namespace using-directives. Use using-declarations instead. [build/namespaces] 

I don't think this repo follows a pattern of writing the CopyRight info in every file.
Also Currently I have used a single line of code `using namespace std;` and hence it shows a linter issue, 
Is there a need to use separate using declarations? 

Kindly review my PR

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**